src/main/java/core/basesyntax/model/Message.java

```diff
@@ -1,8 +1,24 @@
 package core.basesyntax.model;
 
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import org.hibernate.annotations.Cascade;
+import org.hibernate.annotations.CascadeType;
+
+@Entity
+@Table(name = "messages")
 public class Message {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String content;
+    @OneToMany
```

Comment:
Almost sure that isn't mistake. But we can use join rules here cause of bad naming of Hibernate.
Also why do you use (fetch = FetchType.EAGER) in other case but not in this one?